yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1240,22 +1249,24 @@ func packAndSendAnalyzeSchemaPayload(status string, errorMsg string) {
 		if !lo.ContainsBy(reasonsToSendObjectNameToCallhome, func(r string) bool {
 			return strings.Contains(issue.Reason, r)
 		}) {
-			issue.ObjectName = "XXX" // Redacting object name before sending in case reason is not in list
+			issue.ObjectName = constants.OBFUSCATE_STRING // Redacting object name before sending in case reason is not in list
 		}
 		for _, sensitiveReason := range reasonsIncludingSensitiveInformationToCallhome {
-			if strings.Contains(issue.Reason, sensitiveReason) {
-				switch sensitiveReason {
-				case UNSUPPORTED_DATATYPE, UNSUPPORTED_DATATYPE_LIVE_MIGRATION:
-					//e.g. Reason "Unsupported datatype - xml on column - data"
-					//sending only "Unsupported datatype - xml"
-					issue.Reason = strings.Split(issue.Reason, "on column -")[0]
-				default:
-					issue.Reason = sensitiveReason
+			if strings.HasPrefix(issue.Reason, sensitiveReason) {
```

Comment:
pls test this logic manually once with callhome. @priyanshi-yb has a server that you can use to easily test.

Reply:
verified.